Documentation/guides/BuildProcess.md

```diff
@@ -555,7 +562,7 @@ when packaging Release applications.
     * `$(AndroidUseApkSigner)` is `False`.
     * `$(AndroidCreatePackagePerAbi)` is `False`.
 
-[apk]: https://en.wikipedia.org/wiki/Android_application_package]
+[apk]: https://en.wikipedia.org/wiki/Android_application_package
```

Comment:
There is also a brief description of APK files at the beginning of this same document in the #overview and #application-packages sections that could be another option for this link, in case linking to Wikipedia is unusual. The Wikipedia article itself links to <https://developer.android.com/guide/components/fundamentals>, which has a brief two sentence description of APK files. It looks like that is indeed the main place on developer.android.com that defines what an APK file is, but it is quite brief. For example, it doesn't mention that an APK file uses the `.zip` file format.

A related idea could be to update the #application-packages section to include a bit of information about the new Android App Bundle format.

We don't have to decide on any of that for this PR necessarily. We can always come back to it in the future.